ironfish/src/rpc/routes/wallet/burnAsset.test.ts

```diff
@@ -87,6 +87,7 @@ describe('burnAsset', () => {
 
       expect(response.content).toEqual({
         assetId: asset.id().toString('hex'),
+        name: BufferUtils.toHuman(asset.name()),
```

Comment:
we always send this back as hex16 encoded `assetName` then you can toHuman on the other side, or people cannot get the raw asset name buffer.

```suggestion
        name: asset.name().toString('hex'),
```